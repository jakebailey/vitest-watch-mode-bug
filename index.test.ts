import { test } from "vitest";

function range(n: number) {
    const x: number[] = [];
    for (let i = 0; i < n; i++) {
        x.push(i);
    }
    return x;
}

const tests = range(10);

function sleep(ms: number) {
    return new Promise((resolve) => setTimeout(resolve, ms));
}

for (const t of tests) {
    test.concurrent(`test ${t}`, async ({ expect }) => {
        await sleep(500);
        expect(t).toMatchSnapshot();
        expect(t).toMatchSnapshot();
        expect(t).toMatchSnapshot();
    });
}
