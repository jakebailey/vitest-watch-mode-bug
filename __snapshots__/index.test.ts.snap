// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test 0 1`] = `0`;

exports[`test 0 2`] = `0`;

exports[`test 0 3`] = `0`;

exports[`test 1 1`] = `1`;

exports[`test 1 2`] = `1`;

exports[`test 1 3`] = `1`;

exports[`test 2 1`] = `2`;

exports[`test 2 2`] = `2`;

exports[`test 2 3`] = `2`;

exports[`test 3 1`] = `3`;

exports[`test 3 2`] = `3`;

exports[`test 3 3`] = `3`;

exports[`test 4 1`] = `4`;

exports[`test 4 2`] = `4`;

exports[`test 4 3`] = `4`;

exports[`test 5 1`] = `5`;

exports[`test 5 2`] = `5`;

exports[`test 5 3`] = `5`;

exports[`test 6 1`] = `6`;

exports[`test 6 2`] = `6`;

exports[`test 6 3`] = `6`;

exports[`test 7 1`] = `7`;

exports[`test 7 2`] = `7`;

exports[`test 7 3`] = `7`;

exports[`test 8 1`] = `8`;

exports[`test 8 2`] = `8`;

exports[`test 8 3`] = `8`;

exports[`test 9 1`] = `9`;

exports[`test 9 2`] = `9`;

exports[`test 9 3`] = `9`;
